### ISSUE TITLE
Analyze mempool/`estimatesmartfee` data and re-calibrate `normal` feerate estimate parameters

### ISSUE THREAD

**Issue by 0e319b6** [ElementsProject/lightning#4172]
See #4113 .

Previously in IRC @rustyrussell and I agreed it would be useful to have some data on how the mempool and `estimatesmartfee` results change over time in order to figure out the best way to estimate fees.

> I'll be happy to update the value of "normal" too over, depending on what our best guess is from your observations.

The observation data can be found [here](https://github.com/0e319b6/feerate-estimate-data).

**Comment by hosiawak** [comment]
Please see my comments about issues with current bitcoin core feerate estimates [here](https://github.com/ElementsProject/lightning/issues/2797#issuecomment-727012025)

tl;dr Pluggable feerate estimators would be great so we don't have to rely on core's "very interesting" estimates.

**Comment by rustyrussell** [comment]
AI cleanup of old issues.  If wrong, please open a new issue.


Triage decision summary:
Issue requests analysis of mempool/fee estimation data from 2020 to recalibrate fee rate parameters. No activity since November 2020 and appears to be a one-time data collection task rather than an ongoing feature request or bug.
